apps/i18n/applab/en_us.json

```diff
@@ -18,8 +18,120 @@
   "designToolboxHeader": "Design Toolbox",
   "designWorkspaceDescription": "Click on an element or choose it from the dropdown below to edit its properties. Create code in Events.",
   "designWorkspaceHeader": "Workspace",
+  "designWorkspace_copyToScreenButton": "Copy to screen",
+  "designWorkspace_eventInsertButton": "Insert and show code",
+  "designWorkspace_makeDefaultButton": "Make Default",
+  "designWorkspace_restoreThemeButton": "Apply Theme",
+  "designWorkspace_eventsTab": "EVENTS",
+  "designWorkspace_propertiesTab": "PROPERTIES",
+  "designElement_button": "Button",
+  "designElement_button_clickEventDesc": "Triggered when the button is clicked with a mouse or tapped on a screen.",
+  "designElement_canvas": "Canvas",
+  "designElement_canvas_clickEventDesc": "Triggered when the canvas is clicked with a mouse or tapped on a screen.",
+  "designElement_chart": "Chart",
+  "designElement_chart_drawEvent": "drawChart",
+  "designElement_chart_drawEventDesc": "Draws the chart using data you provide.",
+  "designElement_chart_drawFromRecordsEvent": "drawChartFromRecords",
+  "designElement_chart_drawFromRecordsEventDesc": "Draws the chart using App Lab's table data storage.",
+  "designElement_checkbox": "Checkbox",
+  "designElement_checkbox_changeEventDesc": "Triggered when the checkbox state changes both from checked to unchecked and unchecked to checked.",
+  "designElement_dropdown": "Dropdown",
+  "designElement_dropdown_changeEventDesc": "Triggered every time an option is selected from the dropdown.",
+  "designElement_image": "Image",
+  "designElement_image_clickEventDesc": "Triggered when the image is clicked with a mouse or tapped on a screen.",
+  "designElement_label": "Label",
+  "designElement_label_clickEventDesc": "Triggered when the label is clicked with a mouse or tapped on a screen.",
+  "designElement_photoSelect": "Photo Select",
```

Comment:
I wonder if we should add `_` to keys on lines 45 and 46 below for consistency in naming convention.

Reply:
I didn't do it only because changing translation keys will be considered as adding new translations which will start a new translation process for them

Reply:
I believe Artem is right (I would need to test this to prove otherwise), modifying a key would be interpreted as a deletion of the old key and addition of a new key, erasing previous translations in crowding. So it is preferred to not modify keys. 
Now, I notice the inconsistency in the naming convention, and I wonder if we should stick to just Camel Case, Although the new keys feel much more understandable to me.

Reply:
Either convention, I think consistency would be nice. Thanks for the explanation!

Reply:
We just came up to the following conclusion:
- underscore (`_`) is used to indicate hierarchy. (our current system does not support nested hierarchy).
- Old keys will be kept as they are for now. (They will eventually get updated)
- New strings will use the hierarchical convention.

Reply:
Thanks for sharing this!